db/migrate/20220603192202_add_column_to_posts.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :title, :string
+    add_column :posts, :text, :text
+    add_column :posts, :updated_at, :timestamp
```

Comment:
- [ ] I ran into an error when I tried running the migration. The error is pointing to like 5 because updated_at already exists. 

Error Screenshot |
:---------------------:|
![a1](https://user-images.githubusercontent.com/39523312/171939580-8184b961-a902-4af1-b0be-27ef2f3e3e74.png)

You can try removing line 5 to fix the problem